homeassistant/components/rachio/device.py

```diff
@@ -196,3 +259,13 @@ def stop_watering(self) -> None:
         """Stop watering all zones connected to this controller."""
         self.rachio.device.stop_water(self.controller_id)
         _LOGGER.info("Stopped watering of all zones on %s", str(self))
+
+    def pause_watering(self, duration) -> None:
+        """Pause watering on this controller."""
+        self.rachio.device.pause_zone_run(self.controller_id, duration * 60)
+        _LOGGER.debug("Paused watering on %s for %s minutes", str(self), duration)
```

Comment:
We don't need to copy to string since the class has a `__str__` magic method.

Reply:
fixed